main.go

```diff
@@ -208,14 +206,6 @@ func run(opts *options) error {
 	}
 	version.PrintVersionLogs(setupLog)
 
-	if !opts.v2APIEnabled {
```

Comment:
Can we edit this and replace check with `webhookEnabled` so we keep message about 1.8.0?

Reply:
👍